scripts/train_alternating.py

```
@@ -0,0 +1,281 @@
+        ]
+    )
+    return logging.getLogger(__name__)
+
+
+def create_cycle_iterator(dataloader):
+    while True:
+        for batch in dataloader:
+            yield batch
+
+
+def compute_losses(model, batch, device, logger, lambda_weight=1000.0, task_type: str = "graph"):
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)